prebuild/index.js

```diff
@@ -55,25 +59,90 @@ function prebuildify () {
 }
 
 function prebuildTarget (arch, target) {
+  if (NAPI_RS === 'true' && platform === 'linux' && arch === 'ia32') return
+
   if (platform === 'linux' && arch === 'ia32' && semver.gte(target.version, '14.0.0')) return
   if (platform === 'win32' && arch === 'ia32' && semver.gte(target.version, '18.0.0')) return
 
-  const output = `prebuilds/${platform}${libc}-${arch}/node-${target.abi}.node`
-  const cmd = [
-    'node-gyp rebuild',
-    `--target=${target.version}`,
-    `--target_arch=${arch}`,
-    `--arch=${arch}`,
-    `--devdir=${cache}`,
-    '--release',
-    '--jobs=max',
-    '--build_v8_with_gn=false',
-    '--v8_enable_pointer_compression=""',
-    '--v8_enable_31bit_smis_on_64bit_arch=""',
-    '--enable_lto=false'
-  ].join(' ')
+  let cmd;
+
+  if (NAPI_RS === 'true') {
+    let build_target;
+    let rust_env_flags = "";
+    switch (platform) {
```

Comment:
This entire switch (and sub-switches) would be clearer with a map.

Reply:
done